android/analytics.md

```diff
@@ -1,8 +1,8 @@
 <br />
 
 **Note**
-Amplify Android is in preview mode and not intended for production usage at this time. We welcome feedback to improve your experience in using Amplify Android.
-[Click here](../sdk/android/analytics) to access the Analytics category guide for Android SDK 2.0 docs.
+This guide shows how to build an app using our Amplify Libraries for Android (Preview) and the Amplify CLI toolchain.
+To use the existing AWS Mobile SDK for iOS instead, [click here.](../sdk/android/analytics)
```

Comment:
do we want to explain the difference here between the SDK and Amplify? I think that's still confusing for customers.

Reply:
I think it's a fair expectation. Since Amplify is in preview it would not be an apples to apples comparison. In addition, the note may not be enough space to explain the difference. However, we can add the difference to Getting started pages. Thoughts?